packages/eslint-plugin/docs/rules/adjacent-overload-signatures.md

```diff
@@ -86,10 +86,16 @@ export function foo(sn: string | number): void;
 
 If you don't care about the general structure of the code, then you will not need this rule.
 
-## Compatibility
+## Related To
 
 - TSLint: [adjacent-overload-signatures](https://palantir.github.io/tslint/rules/adjacent-overload-signatures/)
 
+## Source
```

Comment:
Suggestion: can this be a `###` h3 underneath `## Attributes`?

Reasoning: I think the source code for a rule would be the last, least significant thing most end users would want to see, while developers are accustomed to having it near the bottom of the page.

Reply:
Done.